.github/workflows/build.yml

```diff
@@ -158,6 +158,10 @@ jobs:
         working-directory: ./tooling/Scripts/
         run: ./PackEachExperiment.ps1 -date ${{ env.VERSION_DATE }}${{ env.VERSION_PROPERTY != '' && format(' -postfix {0}', env.VERSION_PROPERTY) || '' }}
 
+      - name: Validate package names
+        if: ${{ env.VERSION_PROPERTY != '' }}
```

Comment:
Should we always run this? Or throw earlier above if this env isn't set? As we should always expect it to be set, eh?

Reply:
This check is isn't to make sure `VERSION_PROPERTY` doesn't go missing it's to make sure any nuget packages produced have the version property if it's already set. The `VERSION_PROPERTY` is always set in labs. I added the `if:` check so it can be copied and used in other repos.